types/d3-array/d3-array-tests.ts

```diff
@@ -208,6 +212,9 @@ numOrUndefined = d3Array.min(mixedObjectArray, accessorMixedObjectToNumOrUndefin
 strOrUndefined = d3Array.min(mixedObjectArray, accessorMixedObjectToStrOrUndefined);
 numOrUndefined = d3Array.min(readonlyMixedObjectOrUndefinedArray, accessorReadOnlyMixedObjectToNumOrUndefined);
 
+let minIndex: number = d3Array.minIndex([3, 3, 1, 1]); // 2
+minIndex = d3Array.minIndex(["20", "3"]); // 0
```

Comment:
In some places `numOrUndefined` is used. Similarly, we also have [num](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47181/files#diff-e8b9f215cbab373580660b51a4326b3bR42). Was thinking that we could possibly avoid redeclaring `minIndex` for consistency.